File: Pcbc.Domain/ViewModels/AudioFileViewModel.cs
namespace Pcbc.Domain.ViewModels
{
    /// <summary>
    ///     Represents an audio file
    /// </summary>
    public class AudioFileViewModel
    {
        public string FileName { get; set; }
        public string FullFileName { get; set; }
        public string FileExtension { get; set; }
        public string NetworkPath { get; set; }
    }
}

File: Pcbc.Sermons/ClientApp/src/app/view-models/audio-file-view-model.ts

export class AudioFileViewModel {

  public fileName: string;
  public fullFileName: string;
  //the extension of the file like ".mp3"
  public fileExtension: string;
  //network path to the file being served, like "\sermons\Gospel of Mark\Servant of All.m4a"
  public networkPath: string;

  constructor() {
    this.fileName = "";
    this.fullFileName = "";
    this.fileExtension = "";
    this.networkPath = "";
  }
}
